io/io/src/TStreamerInfoReadBuffer.cxx

```diff
@@ -326,15 +326,21 @@ Int_t TStreamerInfo::ReadBufferSkip(TBuffer &b, const T &arr, const TCompInfo *c
       }
 
       // skip Class *  not derived from TObject with comment field  //->
-      case TStreamerInfo::kSkip + TStreamerInfo::kAnyp: {
+      case TStreamerInfo::kSkip + TStreamerInfo::kAnyp:
+      case TStreamerInfo::kSkip + TStreamerInfo::kAnyp + TStreamerInfo::kOffsetL:
+      {
          DOLOOP {
-            b.SkipObjectAny();
+            for (Int_t j=0;j<compinfo->fLength;j++) {
+               b.SkipObjectAny();
+            }
```

Comment:
I think this is not tested by https://github.com/root-project/roottest/pull/1247. In this case, can a test be added?

Reply:
It is.  See line 49 in  execSourceTypes.cxx in https://github.com/root-project/roottest/pull/1247